front/public/locales/en/translation.json

```
@@ -159,7 +159,9 @@
         "error": "An error occurred while loading the data",
         "filter-level": "Error level",
         "filter-type": "Error type",
-        "total-error": "{{count}} errors found",
+        "total-errors": "{{count}} errors found",
+        "total-all": "{{count}} errors and warnings found",
+        "total-warnings": "{{count}} warnings found",
         "total-error_one": "One error found",
         "total-error_zero": "No error found",
```

You need to add keys _one and _zero for the 2 new keys. You also have to update the existing keys
```suggestion
        "total-errors": "{{count}} errors found",
        "total-all": "{{count}} errors and warnings found",
        "total-warnings": "{{count}} warnings found",
        "total-errors_one": "One error found",
        "total-errors_zero": "No error found",
```